ewebsock/src/web.rs

```diff
@@ -150,12 +150,14 @@ pub(crate) fn ws_connect_impl(
     {
         let on_event = on_event.clone();
         let onerror_callback = Closure::wrap(Box::new(move |error_event: web_sys::ErrorEvent| {
-            log::error!(
-                "error event: {}: {:?}",
-                error_event.message(),
-                error_event.error()
-            );
-            on_event(WsEvent::Error(error_event.message()));
+            let message = js_sys::Reflect::get(&error_event, &"message".into()).unwrap();
+            let error = js_sys::Reflect::get(&error_event, &"error".into()).unwrap();
```

Comment:
We should probably replace these `unwrap`s with a fallback, e.g. using `unwrap_or_default`, otherwise we just get another crash at another time

Reply:
You are right. I just changed to unwrap_or_default.
I do not think it could actually crash, but it looked ugly.